content/pt-br/multitenancy.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Multitenancy
+status: Completed
+category: propriedade
+---
+
+## O que é
+
+Multitenancy (em inglês, multi-inquilinos) se refere a uma única instalação de software que atende a múltiplos inquilinos. Um inquilino pode ser um usuário, uma aplicação ou um grupo de usuários/aplicações que usam esse software para operar no seu próprio dataset. Esses inquilinos não compartilham dados (a não ser que explicitamente configurado pelo proprietário), e podem até não estar cientes um dos outros.
```

Comment:
"Multitenancy (em inglês, multi-inquilinos)" is an English word. I think the literal translation isn't necessary.

Reply:
I agree with that, I'll be better to the users search for tenancy instead of "inquilinos" in Portuguese.
My suggestion here is to use the original word that is more easy to understand the concept that we are talking about.

Reply:
+1
From what I researched seems like either `multitenancy` or `multi-tenant` is used directly, without translation (e.g. https://www.treinaweb.com.br/blog/o-que-e-uma-aplicacao-multitenant , https://eximia.co/single-tenant-multi-tenant-ou-hibrido-que-estrategia-adotar-para-aplicacoes-que-atendem-diversos-clientes/)

Reply:
Either way seems fine for me. What I thought was to associate the concept of of multitenancy to 'inquilino' as soon as possible so that the rest of the text would be more connected.